### PR TITLE
[DMCA] Add copyright headers & ReadMe statements

### PR DESCRIPTION
**Pull request by lasley**
* Add copyright headers on files that have not been entirely rewritten
* Add modified work statements on modules that evolved from GNUHealth

I did not alter copyright on files that are completely rewritten, as it cannot be asserted in that case.

Modified work statements are not included in ReadMes for modules that are entirely new works, as they are combined works that are subject to GPL due to dynamic linking but not modified works.

### CODE REVIEW

**Comment by lasley** [comment]
@OCA/board - I'm deleting v9 branch so we can keep this repo alive. I unfortunately don't have the time to get it fixed in the 24 hour window, so it's the best option I think.

File: medical_disease/models/medical_pathology.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # -*- coding: utf-8 -*-
 # #############################################################################
 #
-# Tech-Receptives Solutions Pvt. Ltd.
-# Copyright (C) 2004-TODAY Tech-Receptives(<http://www.techreceptives.com>)
-# Special Credit and Thanks to Thymbra Latinoamericana S.A.
+# Copyright 2008 Luis Falcon <lfalcon@gnusolidario.org>
```
Comment (elicoidal) [inline medical_disease/models/medical_pathology.py:4]: Question: did you validate with him the date and copyright information?

**Review by elicoidal — APPROVED** [review]
LGTM with one small question

File: medical_disease/models/medical_pathology.py (hunk 1 above)
Comment (lasley) [reply medical_disease/models/medical_pathology.py:4]: They will not speak to us like human beings unfortunately. This is the date of inception for GNUHealth, so it's the date I put.

Comment (lasley) [reply medical_disease/models/medical_pathology.py:4]: (It's the info here too- https://sourceforge.net/projects/medical/files/Oldfiles/1.0.1)

File: medical/README.rst [hunk 2]
```diff
@@ -105,6 +105,33 @@ Contributors
 * Brett Wood <bwood@laslabs.com>
 * Jordi Ballester Alomar <jordi.ballester@eficent.com>
 
+The current project as it is today represents an evolution of the original work
+from from started from Luis Falcon. See https://sourceforge.net/projects/medical/files/Oldfiles/1.0.1,
```
Comment (pedrobaeza) [inline medical/README.rst]: Double `from` (several occurrences)

**Review by pedrobaeza — DISMISSED** [review]

**Review by jgrandguillaume — APPROVED** [review]
Just correct the double "from from" spotted by Pedro

**Pull request merged**